provisioning/service/src/Auth/ServiceAuthenticationWithSharedAccessPolicyToken.cs

```diff
@@ -20,20 +20,20 @@ internal sealed class ServiceAuthenticationWithSharedAccessPolicyToken : IAuthen
         /// <param name="token">Token associated with the shared access policy.</param>
         public ServiceAuthenticationWithSharedAccessPolicyToken(string policyName, string token)
         {
-            this.SetPolicyName(policyName);
-            this.SetToken(token);
+            SetPolicyName(policyName);
+            SetToken(token);
         }
 
         public string PolicyName
         {
-            get { return this.policyName; }
-            set { this.SetPolicyName(value); }
+            get { return policyName; }
+            set { SetPolicyName(value); }
```

Comment:
```suggestions
get => polcyName;
set => SetPolicyName(value);
```

Reply:
Already taken care of in the second round of cleanup.